.github/workflows/test.yml

```diff
@@ -21,11 +26,13 @@ jobs:
         python -m pip install --upgrade pip
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
         if [ -f test-requirements.txt ]; then pip install -r test-requirements.txt; fi
```

Comment:
If pylint is added to CI then it should be added to test-requirements.txt for installation.